app/components/user_menu/signed_in_component.html.erb

```diff
@@ -22,7 +22,9 @@
     <div data-toggle-target="element" class="hidden origin-top-right absolute right-0 z-10 mt-2 w-48 rounded-md shadow-lg py-1 bg-white ring-1 ring-black ring-opacity-5 focus:outline-none" role="menu" aria-orientation="vertical" aria-labelledby="user-menu-button" tabindex="-1">
       <%= render UserMenu::MenuItemComponent.new(t(".get_started"), new_role_path, condition: neither?) %>
       <%= render UserMenu::MenuItemComponent.new(t(".my_business_profile"), new_business_path, condition: business?) %>
-      <%= render UserMenu::MenuItemComponent.new(t(".my_developer_profile"), new_developer_path, condition: developer?) %>
+      <% if developer? %>
+        <%= render UserMenu::MenuItemComponent.new(t(".my_developer_profile"), developer_path(user.developer), condition: developer?) %>
+      <% end %>
```

Comment:
I don't think this change is necessary because of the `condition:` option, right?